src/DonorDashboards/Tabs/EditProfileTab/AvatarRoute.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Give\DonorDashboards\Tabs\EditProfileTab;
+
+use WP_REST_Request;
+use Give\DonorDashboards\Tabs\Contracts\Route as RouteAbstract;
+use WP_REST_Response;
+
+/**
+ * @unreleased
+ */
+class AvatarRoute extends RouteAbstract {
+
+	/**
+	 * @inheritdoc
+	 */
+	public function endpoint() {
+		return 'avatar';
+	}
+
+	/**
+	 * @inheritdoc
+	 */
+	public function args() {
+		return [];
+	}
+
+	/**
+	 * @inheritDoc
+	 *
+	 * @return array
+	 */
+	public function handleRequest( WP_REST_Request $request ) {
+
+		if ( ! ( is_array( $_POST ) && is_array( $_FILES ) ) ) {
+			return [];
+		}
```

Comment:
This is validation and should provide a specific error.

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5745/commits/18c03e1650004f4d3f1738be6faf68e32a8d37cb